docs/sequence_diagrams/notification-LimitPositionBreach.plantuml

```diff
@@ -0,0 +1,134 @@
+@startuml
+' declate title
+title 10.3. Notification Limit Position Breach Rule Validation
```

Comment:
Title is misleading. I know it's how we have it even in the code, but it's not the correct wording. Because it's about Breaching Threshold Alarm Percentage Limit. Limit Breach is not allowed by the switch at all.

Reply:
This is not about an actual breach - as noted it is not allowed, with a separate process to deal with that use case. This is about a current Limit Position Breach (percentage), with the breach % being defined within the rules.  Limit Position Percentage Breach as defined by the rules.  
Suggest to renamed diagram to "Limit Position Percentage Breach"? 
As mentioned, this is not how it is defined in the code, thus there will be a disconnect between the design and the code.